fbrp/tests/fbrp/runtime/test_conda.py

```diff
@@ -8,84 +7,146 @@
-                     dependencies=["ros-noetic-genpy", {"pip" : ["scipy", "cuda110", "pytorch"]}])
-        b = CondaEnv(channels=["pytorch", "nvidia"], dependencies=["pytorch", "numpy", "dataclasses"])
+        a = CondaEnv(
+            channels=["conda-forge", "robostack", "pytorch"],
+            dependencies=["ros-noetic-genpy", {"pip": ["scipy", "cuda110", "pytorch"]}],
+        )
+        b = CondaEnv(
+            channels=["pytorch", "nvidia"],
+            dependencies=["pytorch", "numpy", "dataclasses"],
+        )
         c = CondaEnv.merge(a, b)
-        self.assertTrue(len(c.channels) == 4)
-        self.assertListEqual(c.channels, sorted(["conda-forge", "robostack", "pytorch", "nvidia"]))
-        self.assertTrue(len(c.dependencies) == 5)
+        assert len(c.channels) == 4
+        assert c.channels == sorted(["conda-forge", "robostack", "pytorch", "nvidia"])
+        assert len(c.dependencies) == 5
         ref_deps = sorted(["pytorch", "ros-noetic-genpy", "numpy", "dataclasses"])
         list_deps = []
         for dep in c.dependencies:
             if type(dep) == dict:
-                self.assertListEqual(sorted(dep["pip"]), sorted(["scipy", "cuda110", "pytorch"]))
+                assert sorted(dep["pip"]) == sorted(["scipy", "cuda110", "pytorch"])
             else:
-                self.assertTrue(dep in ref_deps)
+                assert dep in ref_deps
                 list_deps.append(dep)
-        self.assertFalse("pip" in c.dependencies)
+        assert "pip" not in c.dependencies
         c.fix_pip()
-        self.assertTrue("pip" in c.dependencies)
+        assert "pip" in c.dependencies
 
 
 class TestLauncher(IsolatedAsyncioTestCase):
     @patch("builtins.open", new_callable=mock_open, read_data="env_var=data" + "\0")
     @patch("argparse.Namespace")
     async def test_activate_conda_env(self, mock_namespace, mock_file):
-        proc_def = ProcDef(name="test_conda", root=None, rule_file=None, runtime="BaseRuntime", cfg={}, deps=[], env={})
-        launcher = Launcher(name="test_conda", run_command=["python3", "alice.py"], proc_def=proc_def, args=mock_namespace)
+        proc_def = ProcDef(
+            name="test_conda",
+            root=None,
+            rule_file=None,
+            runtime="BaseRuntime",
+            cfg={},
+            deps=[],
+            env={},
+        )
+        launcher = Launcher(
+            name="test_conda",
+            run_command=["python3", "alice.py"],
+            proc_def=proc_def,
+            args=mock_namespace,
+        )
         os_env_patch = mock.patch.dict(os.environ, {"my_path": "path"})
         os_env_patch.start()
         conda_env = await launcher.activate_conda_env()
         mock_file.assert_called_with(f"/tmp/fbrp_conda_test_conda.env")
-        self.assertTrue(len(conda_env) == 1)
-        self.assertDictEqual(conda_env, {"env_var" : "data"})
+        assert len(conda_env) == 1
```

Comment:
curious, why the change from `self.assertTrue` to `assert` ?

Reply:
I "believe" `self.assertTrue` is part of the built-in unittest library. The pytest module, that we're using to run the tests uses and recommends `assert`: https://docs.pytest.org/en/6.2.x/